src/main/java/io/vertx/core/http/impl/HttpClientRequestBase.java

```diff
@@ -35,18 +35,21 @@
   protected final String uri;
   protected final String path;
   protected final String host;
+  protected final int port;
   protected final String query;
   private Handler<Throwable> exceptionHandler;
   private long currentTimeoutTimerId = -1;
+  private long currentTimeoutMs;
   private long lastDataReceived;
-  protected boolean exceptionOccurred;
+  protected Throwable exceptionOccurred;
```

Comment:
Is renaming possible? I know the class could be extended but it's an internal class at the same time